doc/en/user/source/extensions/geofence-server/rest.rst

```diff
@@ -99,40 +99,46 @@ All filter parameters are optional.
      - Used for paging a list of rules. Specifies the number of entries per page. Leave out for no paging. If specified, ``page`` should also be specified.
    * - userName
      - string
-     - Filter rules on username (excludes all other specified usernames).
+     - Filter rules on username (excludes all other specific usernames).
    * - userAny
      - 0 or 1. 
-     - Specify whether rules for 'any' username are included or not.
+     - Specify whether rules matching any username are included or not.
    * - roleName
      - string
-     - Filter rules on rolename (excludes all other specified rolenames).
+     - Filter rules on rolename (excludes all other specific rolenames).
    * - roleAny
      - 0 or 1. 
-     - Specify whether rules for 'any' rolename are included or not.
+     - Specify whether rules matching any rolename are included or not.
+   * - ipaddress
+     - string
+     - Filter rules on IP address range (only select rules with an address range that includes the passed IP address).
+   * - ipaddressAny
```

Comment:
Other entries seems to be using camel case, so I'd suggest using:

* ipAddress
* ipAddressAny